src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -488,8 +505,14 @@ private unsafe int HandleEventReceive(ref RECEIVE data)
         data.TotalBufferLength = totalCopied;
         return (_receiveBuffers.HasCapacity() && Interlocked.CompareExchange(ref _receivedNeedsEnable, 0, 1) == 1) ? QUIC_STATUS_CONTINUE : QUIC_STATUS_SUCCESS;
     }
+
```

Comment:
```suggestion
```

Reply:
why? methods should be separated by line IMHO. So as empty line should follow closing brace.
I think we should do separate style pass on the file. @rzikm seems to have best editor setting....